arch/esp32/esp32.ini

```diff
@@ -45,9 +45,9 @@ lib_deps =
   ${environmental_base.lib_deps}
   ${radiolib_base.lib_deps}
   https://github.com/meshtastic/esp32_https_server.git#23665b3adc080a311dcbb586ed5941b5f94d6ea2
-  h2zero/NimBLE-Arduino@^1.4.2
+  h2zero/NimBLE-Arduino@^1.4.3
   https://github.com/dbinfrago/libpax.git#3cdc0371c375676a97967547f4065607d4c53fd1
-  lewisxhe/XPowersLib@^0.2.6
+  lewisxhe/XPowersLib@^0.2.7
```

Comment:
This is one to watch out for sometimes. In the past there have issues with API changes due to method spelling / renames

Reply:
Ah! That one! Thank you for the reminder.